doc/source/whatsnew/v1.4.0.rst

```diff
@@ -967,6 +967,7 @@ I/O
 - Bug in :func:`read_json` raising ``ValueError`` when attempting to parse json strings containing "://" (:issue:`36271`)
 - Bug in :func:`read_csv` when ``engine="c"`` and ``encoding_errors=None`` which caused a segfault (:issue:`45180`)
 - Bug in :func:`read_csv` an invalid value of ``usecols`` leading to an unclosed file handle (:issue:`45384`)
+- Bug in :func:`read_parquet` when ``engine="pyarrow"`` which caused partial write to disk when column of unsupported datatype was passed (:issue:`44914`)
```

Comment:
can you move to 1.5 (same section)

Reply:
Thanks, I have moved the what's new point to v1.5 and merged with main branch